src/SlowSlothBudget.Web/Startup.cs

```diff
@@ -34,6 +34,8 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
 
+
+            // BART: why is this development auth needed? In theory it only sets some artifical nameidentifier in claims Identity, cant the same be achieved by setting some test user in your current 0auth IDP?
```

Comment:
I now remember why it's needed. Auth0 being an ISP is an online service. I wanted to be able to develop while being offline, hence this hacky dev auth.